src/Serilog.Enrichers.CorrelationId.Standard/Serilog.Enrichers.CorrelationId.Standard.csproj

```diff
@@ -4,11 +4,18 @@
     <TargetFramework>netstandard2.0</TargetFramework>

```

Comment:
Can't we multitarget frameworks using this bit now? Do we need both CSProj?
`<TargetFrameworks>netstandard2.0;v4.5</TargetFramework>`